src/styles/Login.css

```diff
@@ -0,0 +1,57 @@
+/*-------------------------Login page(card)---------------*/
+
+.login-check:invalid {
+	border: var(--Mak-colors-red-500) solid 1px;
+	border-radius: 1.5rem;
+}
+/* .login-check:valid {
+	border: var(--Mak-colors-green-500) solid 1px;
+	border-radius: 1.5rem;
+} */
```

Comment:
Remove this and also check all the files. Remove commented code from all of them

Reply:
Okay, I'll do that.